docs/stencil-docs/reference-docs/theme-objects.md

```diff
@@ -69,15 +69,14 @@ To use dynamic data from the template in client-side code, use the inject helper
 
 <!-- ... -->
 
-<script>
-    var jsContext = JSON.parse({{jsContext}});
-</script>
+var jsContext = JSON.parse({{jsContext}});
+
 ```
 **client code:**
 
 ```js
 // client-side code
```

Comment:
Consider specifying where this code is. Example:

`// assets/js/theme/category.js`